### PR TITLE
Move hint to separate component

### PR DESCRIPTION
**Pull request by fungjj92**
## Overview

The quiz hint will animate in. This will be easier if the hint is it's own component. This PR sets up the hint for animation by Design.

Connects #124 

### Demo

hint

hint2
^ giphy took a weird screencapture, but the images looked normal i swear.

## Notes
I did not move the hint button to a new component. I don't believe it will get animated/anything special that would benefit from being a separate component.

## Testing Instructions
Netlify is sufficient.
Play the quiz and interact with the hint. Try to get questions wrong and see the hint at work.

### CODE REVIEW

File: src/app/src/App.js [hunk 1]
```diff
@@ -43,6 +43,7 @@ import {
 library.add(
     faInfoCircle,
     fasInfoCircle,
+    faExclamationCircle,
```
Comment (kellyi) [inline src/app/src/App.js:46]: Out of curiosity, what does this change do?

File: src/app/src/components/QuizHint.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+const StyledQuizHint = styled(Box)`
+    background: ${themeGet('colors.greens.3')};
+    text-align: center;
+    width: fit-content;
+    margin-left: auto;
+`;
+
+const QuizHint = ({ hint }) => {
+    return (
```
Comment (kellyi) [inline src/app/src/components/QuizHint.js]: Is it ever possible for `hint` to be undefined here? If not, consider adding an `.isRequired` proptype for it and dropping the `{ return` in favor of `({ hint }) => (<StyledQuizHint ....`.

If so, consider adding either a default prop for hint or handling it inline here.

**Review by kellyi — APPROVED** [review]
:+1: tried out the Netlify build and I see the hints there. Made some inline comments on error handling and default props etc; feel free to implement or not

File: src/app/src/App.js (hunk 1 above)
Comment (fungjj92) [reply src/app/src/App.js:46]: We have to individually import each font awesome icon we want to use ~ this one pulls in the (!) icon

File: src/app/src/components/QuizHint.js (hunk 2 above)
Comment (fungjj92) [reply src/app/src/components/QuizHint.js]: Good idea! There will always be a hint.

**Pull request merged**